gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=1541fa36599e12857140465f3c91a97409b4512501c26f9631fb113e392c5bd1
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.1-bin.zip
+distributionSha256Sum=31c55713e40233a8303827ceb42ca48a47267a0ad4bab9177123121e71524c26
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.2-bin.zip
```

Comment:
When there's Gradle update it should be done with the [`wrapper` task](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper) so that `gradlew` and `gradlew.bat` files are updated as well.

In this case it would be `./gradlew wrapper --gradle-version 8.10.2 --distribution-type bin --gradle-distribution-sha256-sum 31c55713e40233a8303827ceb42ca48a47267a0ad4bab9177123121e71524c26`.

Reply:
Thanks for the tip. I will do that and fix the lint errors from the build.